src/Operation/Delete.php

```diff
@@ -176,6 +185,23 @@ public function getCommandDocument(Server $server)
         return $cmd;
     }
 
+    /**
+     * Create options for constructing the bulk write.
+     *
+     * @see https://php.net/manual/en/mongodb-driver-bulkwrite.construct.php
+     * @return array
+     */
+    private function createBulkWriteOptions()
```

Comment:
```suggestion
     */
    private function createBulkWriteOptions(): array
```

As new methods are introduced, we can switch to using return type hints. Eventually, this will be done across the entire project (we have some backlog issues to address that).